src/plugins/Logging.ts

```diff
@@ -1,67 +1,87 @@
+import { LOG_LEVELS } from '../types';
+import dispatchEvent from '../utils/dispatchEvent';
+import onEvent from '../utils/onEvent';
 import GenericPlugin from './GenericPlugin';
 
-const BASE_MESSAGE = '[DEBUG]';
+const BASE_MESSAGE = '[ADJS]';
 
-let adId = 0;
+const colorMap: { [key: string]: string } = {
+  AutoRefresh: 'blue',
+  AutoRender: 'purple',
+  Logging: 'red',
+  Responsive: 'green',
+  Sticky: 'brown',
+  DFPNetwork: 'gray',
+};
 
 class LoggingPlugin extends GenericPlugin {
   public onCreate() {
-    const id = ++adId;
-    (window as any)[`ad${id}`] = this.ad;
+    const { ad } = this;
+    const { id } = ad;
 
+    (window as any)[`ad${id}`] = ad;
     console.log(
       BASE_MESSAGE,
       'Ad Instantiated and assigned as',
-      `window.ad${adId}`,
+      `window.ad${id}`,
     );
+
+    onEvent(id, (event) => {
+      const { level, creator, message, data } = event.detail;
+      // @ts-ignore
+      console[level](
+        `%c${BASE_MESSAGE} ${creator} ${message} ${data || ''}`,
```

Comment:
let's wrap creator in brackets too.
`%c${BASE_MESSAGE}[${creator}] ${message} ${data || ''}`